packages/protocol/contracts/identity/Escrow.sol

```diff
@@ -69,27 +79,70 @@ contract Escrow is
   // Maps senders' addresses to a list of sent escrowed payment IDs.
   mapping(address => address[]) public sentPaymentIds;
 
+  // Maps payment ID to a list of issuers whose attestations will be accepted.
+  mapping(address => address[]) public trustedIssuersPerPayment;
+
+  // Governable list of trustedIssuers to set for payments by default.
+  address[] public defaultTrustedIssuers;
+
+  /**
+   * @notice Sets initialized == true on implementation contracts
+   * @param test Set to true to skip implementation initialization
+   */
+  constructor(bool test) public Initializable(test) {}
+
   /**
    * @notice Returns the storage, major, minor, and patch version of the contract.
    * @return The storage, major, minor, and patch version of the contract.
    */
   function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
-    return (1, 1, 1, 2);
+    return (1, 2, 0, 0);
```

Comment:
IIUC, this upgrade should provide new functionality, but can still be used as before (e.g. it is fully backwards compatible). If that is true, you may want the version to be `1.1.2.0` since the extended semver format we are using here is `(STORAGE, MAJOR, MINOR, PATCH)`. Of course, you don't _need_ a breaking change to bump the major version, and this change is large enough to justify doing so.

Reply:
Yeah so the bump to `MAJOR` is actually due to the upgrade to `UsingRegistryV2`, which changes the ABI in that there is no longer a `setRegistry` method :-/